### ISSUE TITLE
Usage with Frigate

### ISSUE THREAD

**Issue by poxin13** [rb2k/sqwrlmodel#1]
Hello, I realize this is an older repo but I've been seeing some things on reddit recently about squirrel detection and this repo came up. Do you know if it's possible to use with frigate? https://deploy-preview-4055--frigate-docs.netlify.app/configuration/objects/

I literally just stood up my frigate install a few moments ago, very new to this, but the file extensions seem to match. Been playing around with it but can't seem to get it going.

**Comment by rb2k** [comment]
I think I tried but there were some issues with dimensions, but that has been quite a while ago. Haven't tried recently. I currently just use the 'default' model with Frigate.

**Comment by poxin13** [comment]
What do you my bean by default model? As in items from this list? https://docs.frigate.video/configuration/objects/ Unfortunately nothing for squirrel.

**Comment by rb2k** [comment]
> What do you my bean by default model? As in items from this list? https://docs.frigate.video/configuration/objects/ Unfortunately nothing for squirrel.

Yes, I just map the various animals to an “animal” label. Tends to capture most of em

![IMG_0160](https://github.com/rb2k/sqwrlmodel/assets/9519/ed223fa0-c162-4941-9cd4-664bd7d3d61b)

**Comment by poxin13** [comment]
Would you be able to share you configuration for that? Appreciate the help
